### PR TITLE
Add sysid for EVERYTHING

### PR DESCRIPTION
**Pull request by Superbro525Alt**

### CODE REVIEW

**Review by spacey-sooty — CHANGES_REQUESTED** [review]
Debugging stuff should be removed before merge 🙂

**Comment by Superbro525Alt** [comment]
nuh uh

**Comment by Superbro525Alt** [comment]
merge now

**Review by spacey-sooty — CHANGES_REQUESTED** [review]
You've committed multiple gitignored files remove them

File: src/main/java/frc/robot/Robot.java [hunk 1]
```diff
@@ -52,6 +55,23 @@ public Robot() {
...
+
+    m_sysid.add(m_climber::sysIdDynamic);
+    m_sysid.add(m_climber::sysIdQuasistatic);
+
+    m_sysid.add(m_intake::sysIdDynamic);
+    m_sysid.add(m_intake::sysIdQuasistatic);
+
+    m_sysid.addAll(m_drivetrain.getSysIdCommands());
+
+    m_sysid.getAllCommands();
```
Comment (spacey-sooty) [inline src/main/java/frc/robot/Robot.java:73]: What does this do?

**Review by spacey-sooty — CHANGES_REQUESTED** [review]
add back swerve docs

File: src/main/java/frc/robot/Robot.java (hunk 1 above)
Comment (Superbro525Alt) [reply src/main/java/frc/robot/Robot.java:73]: it adds the functions to the class and the method returns a sequential command that runs all of the sysid things for all the subsystems added in forward and reverse

Comment (spacey-sooty) [reply src/main/java/frc/robot/Robot.java:73]: Surely the Sequential Command should be used somewhere

**Comment by spacey-sooty** [comment]
How long does the Command to run? It could be nice to have a SysID auto for a practice match.

**Pull request closed without merging**